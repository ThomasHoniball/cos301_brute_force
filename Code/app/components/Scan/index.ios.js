import Scan from './index.js'

export default Scan