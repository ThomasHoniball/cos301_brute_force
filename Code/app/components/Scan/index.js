import React, { Component } from 'react'
import Styles from './styles';
import { StyleSheet, Text, View } from 'react-native';

class Scan extends Component {
    	render() {
            return (   
            <Text >Barcode Page</Text>
            )
        }
    }
export default Scan