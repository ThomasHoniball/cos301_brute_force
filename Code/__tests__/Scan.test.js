import React from 'react';
import renderer from 'react-test-renderer';

import Scan from '../app/components/Scan';


describe('<Scan />', () => {
	it('renders correctly', () => {
		const tree = renderer.create(<Scan />).toJSON();
		expect(tree).toMatchSnapshot();
	});
	

});
