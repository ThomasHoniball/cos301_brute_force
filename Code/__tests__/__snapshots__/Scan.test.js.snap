// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scan /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#000",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Requesting for camera permission
  </Text>
</View>
`;
